### PR TITLE
refactor reponsive  mobile, print mode, data experience

### PR DESCRIPTION
**Pull request by LuuTheNguyen**
# Refactor reponsive  mobile, print mode, data experience

### Description
  -  Refactor reponsive  mobile
  -  Refactor print mode
  -  Refactor data experience
### Preview
  - Refactor reponsive  mobile: https://i.imgur.com/X1QnKEw.png
  -  Refactor print mode: https://i.imgur.com/7jxB8RY.png
  -  Refactor data experience: https://i.imgur.com/Mqk751l.png

### Checklist
 - [x] Tested on Desktop
 - [x] Tested on Mobile
 - [x] Tested `Print mode`

### CODE REVIEW

**Comment by tietthinh** [comment]
iPad Safari switch theme button break.

File: containers/Home/Contact/Contact.tsx [hunk 1]
```diff
@@ -1,13 +1,41 @@
...
+        case 'skype':
+            return (
+                <span>
+                    <a href={`skype:${content}?chat`}>{content}</a>
+                </span>
+            )
+        default:
+            return <span>{content}</span>
+    }
+}
```
Comment (tietthinh) [inline containers/Home/Contact/Contact.tsx:30]: This is React Component (as defined React.FC) so it should use the React naming convention

<span> tag has been duplicated a alot, can we refactor this? tbh, I think we could simplified the `switch case` solution to object mapping.

File: containers/Home/About/About.tsx [hunk 2]
```diff
@@ -1,14 +1,19 @@
...
         <StyledAbout>
             {about.map((item, index) => (
                 <div key={index}>
-                    <StyledLabel>{item.label}</StyledLabel>{' '}
-                    <StyledContent>{item.content}</StyledContent>
+                    <StyledLabel>{item.label}</StyledLabel>
+                    <StyledContent>
+                        {typeof item.content === 'number'
+                            ? Number(moment().format('YYYY')) - Number(moment(item.content).format('YYYY'))
+                            : item.content}
```
Comment (tietthinh) [inline containers/Home/About/About.tsx:15]: Please separate this into a new variable. Beside, I don't think `typeof item.content === 'number'` is a good solution in the future, but we will refactor this later so this should be good to go

File: containers/Home/Service/Service.tsx [hunk 3]
```diff
@@ -17,50 +17,47 @@ export const Service: React.FC<Props> = ({ service }) => {
...
-                                <span>{item.description}</span>
-                            </div>
-                        )}
-                    </StyledContentService>
+                    <table>
+                        <tbody>
+                            <tr>
+                                <td>
+                                    <StyledHeaderService>
+                                        <StyledHeaderTitleService>{item.time}</StyledHeaderTitleService>
```
Comment (tietthinh) [inline containers/Home/Service/Service.tsx:25]: We should use `moment` in this case, also update the mock data `time` to timestamp (UTC) format.

**Review by peace-bi — APPROVED** [review]

**Review by tietthinh — APPROVED** [review]

**Pull request merged**